pulumi/python/utility/kic-pulumi-utils/setup.py

```diff
@@ -7,5 +7,5 @@
       version_config=True,
       packages=['kic_util'],
       install_requires=[
-          'pyyaml>=5.3.1,<6.0', 'passlib>=1.7.4,<2.0.0', 'GitPython>=3.1.18,<3.2.0'
+          'pyyaml', 'passlib>=1.7.4,<2.0.0', 'GitPython>=3.1.18,<3.2.0'
```

Comment:
**annotation**:
Relaxing this requirement is ok and allows us to update the various pulumi packages which have moved to version >=6